webui/src/lib/api/index.js

```diff
@@ -118,6 +118,36 @@ export class MergeError extends Error {
 // actual actions:
 class Auth {
 
+    async password(token, newPassword, newPasswordConfirm) {
```

Comment:
Why do we need to post both new password and password confirm? It is something the at verify on the client side before you post a request?
There is no need to get to the server side to understand that they are not matched.

Reply:
> Why do we need to post both new password and password confirm? It is something the at verify on the client side before you post a request? There is no need to get to the server-side to understand that they are not matched.

@ortz I think you wanted it to avoid someone abusing the API? but I actually don't think it is necessary.. 

> Method name should describe the action - update password by token - something like that or shorter

I agree this is bad naming, but I'm mirroring the name we have in swagger. should we change both?

Reply:
I think the name in swagger is consistent with the other paths in swagger. For instance /auth/groups, when posting it creates a group, although "groups" doesn't suggest it creates a group.

Reply:
renamed it to updatePasswordByToken

Reply:
If you both think it's preferred not to send it, let's drop it.